### PR TITLE
CRI: add oom_score_adj isolator

### PR DESCRIPTION
**Pull request by squeed**
This is for coreos/rkt#3203
@lucab please review?

### CODE REVIEW

File: schema/types/isolator_linux_specific_test.go [hunk 1]
```diff
@@ -204,3 +204,40 @@ func TestNewLinuxSeccompRetainSet(t *testing.T) {
 		}
 	}
 }
+func TestLinuxOOMScoreAdj(t *testing.T) {
+	tests := []struct {
+		in []string
+
+		wset []LinuxCapability
```
Comment (lucab) [inline schema/types/isolator_linux_specific_test.go]: I guess you forgot to wire in the new tests.

Comment (squeed) [inline schema/types/isolator_linux_specific_test.go]: Oops, this test wasn't needed - I forgot to delete it.

Comment (squeed) [inline schema/types/isolator_linux_specific_test.go]: The codepath tested here doesn't exist for the OOM case, because it's a scalar / simple value.

**Comment by s-urbaniak** [comment]
LGTM

**Comment by jonboulle** [comment]
lgtm

**Pull request merged**